symfony/twig-bundle/6.4/templates/base.html.twig

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>{% block title %}Welcome!{% endblock %}</title>
+        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text></svg>">
```

Comment:
Do you know why we use this sober emoji instead of something more fun like ⭐ or 🎯.

Reply:
It looks like the sf logo... but without sf 😂

Reply:
also, projects must change their favicon to use their own anyway. Having a default one here avoids getting 404 errors when browsers try to load a default `favicon.ico` file otherwise.

Reply:
Oh! Let's add the `sf` then.
```suggestion
        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text><text y=%221.3em%22 x=%220.2em%22 font-size=%2276%22 fill=%22%23fff%22>sf</text></svg>">
```
Chrome
<img width="250" alt="image" src="https://github.com/symfony/recipes/assets/400034/fb643498-24dd-4400-8141-1f467be4a7c4">

Firefox
<img width="233" alt="image" src="https://github.com/symfony/recipes/assets/400034/7bf1968a-4960-4af1-86a4-435bbf237cf8">

Reply:
I like the idea of making a icon from an emoji. It make its very easy to anyone starting a project to use something from the existing panel.